greenpithumb/camera_manager.py

```diff
@@ -3,31 +3,34 @@
 
 logger = logging.getLogger(__name__)
 
+# Format of filename to write for camera image file (assumes timestamp is in
+# UTC), as YYYY-MM-DDTHH:MM:SSZ (ISO 8601 format)
+_FILENAME_FORMAT = '%Y-%m-%dT%H%M%SZ.jpg'
```

Comment:
Oh good point. The encoder in the frontend will still JSONify it with seconds-level precision, but maybe we can address that later.